src/cryptoadvance/specter/wallet.py

```diff
@@ -191,18 +201,24 @@ def from_json(cls, wallet_dict, device_manager, manager, default_alias='', defau
     def get_info(self):
         try:
             self.info = self.cli.getwalletinfo()
-        except:
+            if self.info['keypoolsize'] == 0:
+                self.keypoolrefill(0, end=self.keypool, change=False)
+                self.keypoolrefill(0, end=self.change_keypool, change=True)
```

Comment:
These three lines are making all requests increadibly long.
In Core v19 keypoolsize will be always zero.

Reply:
Hmmm so do you know if there's an alternative there valid for both 19 and 20?